spec/support/unit/reports/shared_examples.rb

```diff
@@ -38,6 +38,12 @@
 # 'ictc' => 'incident creation test case'
 RSpec.shared_examples 'ictc' do |report_label: nil, noop_test: false|
   context "report with #{report_label}" do
+    # All tests cases should have these allow assertions
+    before(:each) do
+      allow(processor).to receive(:time).and_return(Time.now)
```

Comment:
You could also place this somewhere in https://github.com/puppetlabs/puppetlabs-servicenow_reporting_integration/blob/main/spec/unit/reports/servicenow_spec.rb#L10 since it seems like mocking needed by all the tests.

Reply:
👍